test/components/published_dates_test.rb

```diff
@@ -0,0 +1,62 @@
+require 'component_test_helper'
+
```

Comment:
I noticed that other tests check that without input the component doesn't render at all - at the moment yours will render the outer div which is probably undesirable. I tweaked mine to avoid this, you can see my approach here: https://github.com/alphagov/government-frontend/blob/metadata-component/app/views/components/_content-metadata.html.erb - would be good to get @fofr to give an opinion on that approach and apply that or something similar here, and then test for it.